db/Models/util_models.js

```diff
@@ -23,3 +23,12 @@ exports.checkArticleExists= (id) => {
         return rows
     })
 }
+
+exports.checkTopicExists = (topic) => {
+    return db.query("SELECT * FROM articles WHERE topic = $1",[topic]).then(({rows}) => {
```

Comment:
This is checking for articles by a topic, even if the topic exists it will return an empty array because there's no articles attached. 

You should query the topic table instead - to prove it exists